src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -35,13 +35,17 @@ function paymentFromComponent(data, component = {}) {
       data: JSON.stringify(requestData),
       paymentMethod: document.querySelector('#adyenPaymentMethodName').value,
     },
+    async: false,
```

Comment:
This is causing paypal to fail https://github.com/Adyen/adyen-web/issues/1359
any reason why it's needed? @maassenbas